rero_ils/theme/menus.py

```diff
@@ -137,17 +137,30 @@ def init_menu_tools():
         id='ill-request-menu'
     )
 
-    item = current_menu.submenu('main.tool.stats')
+    item = current_menu.submenu('main.tool.stats_billing')
     rero_register(
         item,
-        endpoint='stats.stats',
+        endpoint='stats.stats_billing',
         visible_when=lambda: admin_permission.require().can(),
         text=TextWithIcon(
             icon='<i class="fa fa-money"></i>',
-            text='Statistics'
+            text='Statistics billing'
         ),
         order=20,
-        id='stats-menu'
+        id='stats-menu-billing'
+    )
+
+    item = current_menu.submenu('main.tool.stats_librarian')
+    rero_register(
+        item,
+        endpoint='stats.stats_librarian',
+        visible_when=lambda: librarian_permission.require().can(),
```

Comment:
you can probably use `bool(current_librarian)`

Reply:
If I use bool(current_librarian) I cannot see the 'Statistics librarian' view if I'm logged in as admin.